### PR TITLE
Mop up after #13199

### PR DESCRIPTION
**Pull request by ghost**
This is an attempt to ignore line numbers in native debugger tracebacks. Otherwise, every commit which will touch parts of the runtime will break these tests.

Might not be enough...

(no changes entry needed)

### CODE REVIEW

**Comment by shindere** [comment]
I'm unsure what to do with this PR at the moment?

Should it be reviewed per se right now, or should we let it pending and
wait for a more complete solution proposed by @tcmgilchrist?

**Comment by Octachron** [comment]
Note that the fix is insufficient for #13199 , there is notably some issue on macOS M1, see https://ci.inria.fr/ocaml/job/precheck/985.
@dustanddreams  do you have access to the inria CI, if not please don't hesitate to ask for it.

**Comment by ghost** [comment]
tl;dr: no need to review now.

Long story: @tmcgilchrist has further commits on top of this PR, which make the test oracles even more stable. It's nighttime in his timezone, but I expect (and asked) him to submit another PR with his changes, and I'll clone this one.

**Comment by shindere** [comment]
Plenty of thanks for the clarification, @dustanddreams!

**Comment by tmcgilchrist** [comment]
@Octachron You'll need to do the equivalent of [this](https://github.com/ocaml/ocaml/blob/trunk/.github/workflows/build.yml#L180-L185) on the macOS workers otherwise the test suite won't be able to startup LLDB from a terminal.

**Comment by ghost** [comment]
Superseded by #13485.
